ui/src/parts/progress_arc.tsx

```diff
@@ -29,7 +29,9 @@ const css = stylesheet({
   container: {
     width: '100%',
     height: '100%',
-    position: 'relative',
+    // Height 100% is not working in Safari. Without this, container has 0 height and d3 cannot compute properly.
```

Comment:
Maybe we should get rid of the 100% width/height?

Reply:
Good catch!